frontend/packages/ux-editor/src/containers/FormContext.tsx

```diff
@@ -30,6 +30,14 @@ export const FormContext = createContext<FormContext>({
   debounceSave: undefined,
 });
 
+export const useFormContext = function () {
+  const context = useContext(FormContext);
+  if (context === undefined) {
+    throw new Error('useFormContext must be used within a FormContextProvider.');
+  }
+  return context;
+};
```

Comment:
For en tid tilbake så laget jeg en utils i Dashboard for å opprette en "strict" context. Egentlig for å slippe å re-implementere (og huske å legge til) denne logikken dersom useContext ikke er wrappet i en provoider. 

Ta en titt på denne: https://github.com/Altinn/altinn-studio/blob/master/frontend/dashboard/utils/createStrictContext/createStrictContext.ts


Burde vi flytte den til shared og benytte oss av den som en generell greie for contexts? 🤔

Reply:
Helt enig, veldig god forslag 👍